c/typedb_driver.i

```diff
@@ -47,60 +47,50 @@ struct Type {};
 %extend Type { ~Type() { function_prefix ## _drop(self); } }
 %enddef
 
+%define %dropproxydefined(Type, function_prefix)
+%ignore function_prefix ## _drop;
+%extend Type { ~Type() { function_prefix ## _drop(self); } }
+%enddef
+
 %dropproxy(Error, error)
-%dropproxy(SchemaException, schema_exception)
-%dropproxy(SchemaExceptionIterator, schema_exception_iterator)
 
-%dropproxy(Credential, credential)
-%dropproxy(Options, options)
+//%dropproxy(Credential, credential)
+//%dropproxy(Options, options)
 
-#define connection_drop connection_close
-#define session_drop session_close
+#define typedb_driver_drop connection_close
```

Comment:
eg these should be renamed away from `connection`, which is a term we don't use anymore.